fxrecorder/src/lib/proto.rs

```diff
@@ -100,23 +100,29 @@ impl RecorderProto {
     }
 
     /// Send a resume request to the runner.
-    pub async fn send_resume_request(&mut self) -> Result<(), RecorderProtoError> {
+    pub async fn send_resume_request(
+        &mut self,
+        wait_for_idle: bool,
```

Comment:
Nit: slight preference for this being an enum, but I don't think it's critical.

Reply:
👍